package.json

```diff
@@ -22,18 +22,19 @@
     "node": "0.10.x"
   },
   "devDependencies": {
-    "typescript": "^2.4.1",
-    "@types/node": "7.0.12",
-    "@types/mocha": "^2.2.40",
     "@types/chai": "3.4.35",
+    "@types/github-url-to-object": "^4.0.0",
+    "@types/mocha": "^2.2.40",
```

Comment:
dependabot suggested an uprade to 5.2.5 in PR [#33] (https://github.com/gorkem/penfold/pull/33/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R30)